covid_graphs/covid_graphs/country_graph.py

```diff
@@ -330,25 +348,20 @@ def show_country_plot(country_data_file: Path, prediction_file: Path):
             formula.create_formula_from_proto(atg_parameters)
             for atg_parameters in country_atg_parameters.parameters
         ]
+        start_idx = _get_earliest_displayable_idx(fitted_formulas, MAX_PEAK_DISTANCE)
 
-        fitted_predictions = [
-            # TODO: move this into a function, similar to get_fitted_predictions
-            CountryPrediction(
-                prediction_event=PredictionEvent(
-                    name=f"daily_fit_{formula.last_data_date.strftime('%Y_%m_%d')}",
-                    last_data_date=formula.last_data_date,
-                    prediction_date=formula.last_data_date,
-                ),
-                country=country_report.short_name,
-                formula=formula,
-            )
-            for formula in fitted_formulas
-        ]
+        fitted_predictions = prediction_generator.create_predictions_from_formulas(
+            fitted_formulas[start_idx:], country_report.short_name
+        )
```

Comment:
There is something suspicious about this (maybe for a different PR). We already have generated predictions (loaded from a file), but we kind of create them again.

Reply:
It's because someone didn't fix the prediction DB yet! :smile: